src/dex-ui/pages/dao/MintNFTModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from "react";
+import { Flex, SimpleGrid, Text } from "@chakra-ui/react";
+import { AlertDialog, Button, Color, FormInputList } from "@dex-ui-components";
+import { useFieldArray, useForm } from "react-hook-form";
+import { MintNFTTokensFormData } from "./types";
+import { MirrorNodeTokenById } from "@services";
+
+export interface MintNFTModalProps {
+  token: MirrorNodeTokenById;
+  handleMintNFT?: (tokenLinks: string[]) => void;
+}
+export const MintNFTModal = (props: MintNFTModalProps) => {
+  const { token, handleMintNFT } = props;
+  const [dialogsOpenState, setDialogsOpenState] = useState(false);
+  const defaultFieldValue = { value: "" };
+  const maxSupply = Number(token.data.max_supply);
+  const totalSupply = Number(token.data.total_supply);
+  let fieldButtonText = "+ Add Link";
```

Comment:
Just for Information `total_supply` is = minted + burned `tokens`

Reply:
I checked the flow for burned tokens. total_supply = minted - burned tokens. so if we have max limit of 10. minted 5 tokens and then burned 3, then we can mint 8 more tokens. ( 10 - (5-3)).